java/core/src/main/java/com/google/protobuf/BinaryWriter.java

```diff
@@ -670,7 +670,7 @@ public <K, V> void writeMap(int fieldNumber, MapEntryLite.Metadata<K, V> metadat
     }
   }
 
-  static final void writeMapEntryField(
+  static void writeMapEntryField(
```

Comment:
Although these methods are `static`, I think there is still some value to marking them `final` since that prevents them from being hidden by another static method in a subclass.

Reply:
Thanks for the feedback, @acozzette. I have reverted the change.